glide-loader/src/main/java/org/wordpress/aztec/glideloader/GlideVideoThumbnailLoader.kt

```diff
@@ -39,8 +45,21 @@ class GlideVideoThumbnailLoader(private val context: Context) : Html.VideoThumbn
                         callbacks.onThumbnailFailed()
                     }
 
-                    override fun onResourceReady(resource: GlideDrawable?, glideAnimation: GlideAnimation<in GlideDrawable>?) {
-                        callbacks.onThumbnailLoaded(resource)
+                    override fun onResourceReady(resource: Bitmap?, glideAnimation: GlideAnimation<in Bitmap>?) {
+                        //Upscaling bitmap only for demonstration purposes.
```

Comment:
This PR looks nice @khaykov - I've just a minor comment, maybe nitpicking. I see the same used in the image loader. What do you think to put it in common in a Utils class?

Reply:
Yep, you are right! I'll move it to separate class.